sql/core/benchmarks/DataSourceReadBenchmark-jdk21-results.txt

```diff
@@ -2,437 +2,437 @@
 SQL Single Numeric Column Scan
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.8.0-1017-azure
+OpenJDK 64-Bit Server VM 21.0.6+7-LTS on Linux 6.8.0-1020-azure
 AMD EPYC 7763 64-Core Processor
 SQL Single BOOLEAN Column Scan:           Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-SQL CSV                                           10281          10314          46          1.5         653.6       1.0X
-SQL Json                                           7952           8108         220          2.0         505.6       1.3X
-SQL Json with UnsafeRow                            9090           9092           3          1.7         577.9       1.1X
-SQL Parquet Vectorized: DataPageV1                   82             94          10        192.5           5.2     125.8X
-SQL Parquet Vectorized: DataPageV2                   92             99           8        171.9           5.8     112.3X
-SQL Parquet MR: DataPageV1                         1701           1728          38          9.2         108.2       6.0X
-SQL Parquet MR: DataPageV2                         1594           1607          19          9.9         101.3       6.5X
-SQL ORC Vectorized                                  137            142           6        114.9           8.7      75.1X
-SQL ORC MR                                         1464           1465           2         10.7          93.1       7.0X
-
-OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.8.0-1017-azure
+SQL CSV                                           10229          10256          39          1.5         650.3       1.0X
+SQL Json                                           8374           8384          14          1.9         532.4       1.2X
+SQL Json with UnsafeRow                            9509           9511           3          1.7         604.6       1.1X
+SQL Parquet Vectorized: DataPageV1                   83             93           6        189.4           5.3     123.2X
+SQL Parquet Vectorized: DataPageV2                  208            217           7         75.7          13.2      49.2X
```

Comment:
Apparently, the ratio between DataPage1 and DataPageV2 is changed too big in this case.